@navikt/ds-react/layouts/product-page/Panel.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, useState } from "react";
+import { forwardRef, HTMLAttributes, ReactNode } from "react";
+import cl from "classnames";
+import { Button, Heading } from "../../";
+import { Attachment } from "@navikt/ds-icons";
+import { EtikettInfo } from "nav-frontend-etiketter";
+import { Undertekst } from "nav-frontend-typografi";
+
+export interface PanelProps extends HTMLAttributes<HTMLDivElement> {
+  title: string;
+  icon?: ReactNode;
+  highlight?: boolean;
+  anchor?: string;
+  whiteBackground?: boolean;
+  withPadding?: boolean;
+}
+
+const Panel = forwardRef<HTMLDivElement, PanelProps>(
+  ({ icon, highlight, anchor, title, children, className, ...rest }, ref) => {
+    const [copied, setIsCopied] = useState<boolean>(false);
+    const copyRef = React.createRef<HTMLButtonElement>();
+
+    useEffect(() => {
+      const header = document.getElementById("layout-header");
+      header?.classList.add("navds-layout__header--margin");
+    }, []);
+
+    return (
+      <div
+        ref={ref}
+        className={cl(
+          "navds-layout__panel",
+          icon && "navds-layout__panel--margin",
+          highlight && "navds-layout__panel--highlight",
+          className
+        )}
+        {...rest}
+      >
+        {icon && <div className={"navds-layout__panel-icon"}>{icon}</div>}
+        {anchor && <a id={anchor} />}
+        <Heading size={"xl"} className={"navds-layout__panel-title"} level={2}>
+          {title}
+        </Heading>
+        {anchor && (
+          <div className={"navds-layout__panel-copy"}>
+            <Button
+              ref={copyRef}
+              variant={"secondary"}
+              className={"navds-layout__panel-copy-button"}
+              onClick={() => {
+                setIsCopied(true);
+                setTimeout(() => {
+                  copyToClipboard(`${window.location.href}/#${anchor}`);
+                  setIsCopied(false);
+                }, 1000);
+              }}
+            >
+              <Attachment className={"navds-layout__panel-copy-icon"} />
+              <span>Kopier lenke</span>
```

Comment:
Kopierer dette ankerlenka? Jeg tror det er vanligere at ankerlenker bare er en lenke som tar nettleseren til det ankeret, og at om man vil kopiere lenka så gjør man det i url-feltet. Her er et eksempel på sånn ankerlenker vanligvis fungerer: https://reactjs.org/docs/hooks-effect.html#example-using-classes

Reply:
Hmm, litt usikker på hva du mener her, det er jo riktig det du sier, men det var et ønske å ha funksjonalitet for å kopierer lenke + anker ved en seksjon.

Si at du har scrollet langt ned på siden til til et panel og ønsker å dele informasjonen med andre.
Da klikker man på knappen og kan dele https://www.nav.no/app#seksjon1. 
Folk som får tilsendt den lenken vil hoppe til riktig sted på siden.

Reply:
Godt mulig jeg har misforstått noe her, men på siden du lenket til så kopieres ikke lenka når man trykker på denne knappen:
![image](https://user-images.githubusercontent.com/1894119/110615748-07be7f80-8194-11eb-83d3-fe0b34d859aa.png)
Det som skjer istedenfor er at nettleseren navigerer til urlen lenka peker på, også kan man kopiere lenka fra url-feltet om man vil.

Reply:
A-ha, da skjønner jeg hva du mener :) Ja, dagpenger-teamet løser det litt annerledes, selv om prinsippet er det samme. Vi tenkte at det kanskje er lettere for brukeren om vi kopierer lenka til clipboard, slik at brukeren slipper å kopiere den fra url-feltet. Men absolutt åpen for å endre det :) @chrisarnesen tanker?

Reply:
Jeg legger denne i min TODO.